sklearn_pandas/features_generator.py

```diff
@@ -0,0 +1,50 @@
+def gen_features(columns, classes=None):
+    """Generates a feature definition list which can be passed
+    into DataFrameMapper
+
+    Params:
+
+    columns     a list of column names to generate features for.
+
+    classes     a list of classes for each feature, a list dictionaries with
+                transformer class and init parameters, or None.
+
+                If list of classes is provided, then each of them is
+                instantiated with default arguments:
+
+                    classes = [StandardScaler, LabelBinarizer]
+
+                If list of dictionaries is provided, then each of them should
+                have a 'class' key with transformer class. All other keys are
+                passed into 'class' value constructor:
+
+                    classes = [
+                        {'class': StandardScaler, 'with_mean': False},
+                        {'class': LabelBinarizer}
+                    }]
+
+                If None value selected, then each feature left as is.
+
+    """
+    if classes is None:
+        return [(column, None) for column in columns]
+
+    feature_defs = []
+
+    for column in columns:
+        feature_transformers = []
+
+        for definition in classes:
+            if isinstance(definition, dict):
+                params = definition.copy()
+                klass = params.pop('class')
+                feature_transformers.append(klass(**params))
+            elif isinstance(definition, type):
```

Comment:
I believe you should substitute the `elif` with a simple `else` since we want all other cases to be instantiated even if the class is not a new-style one. See:
```
In [6]: class A:
   ...:     pass
   ...: 

In [7]: isinstance(A, type)
Out[7]: False

In [8]: class B(object):
   ...:     pass
   ...: 

In [9]: isinstance(B, type)
Out[9]: True
```
However we would like to instantiate `A` in this case anyhow. Also, the passed object can be a function that returns an instantiated class as well.

Reply:
Hm, I see, didn't know about it. I guess the reason is I mostly worked with Python v3.x with its default new-style classes. Will fix that.